src/providers/json-rpc-provider.ts

```diff
@@ -106,6 +107,14 @@ export class JsonRpcProvider extends Provider {
         return this.sendJsonRpc('validators', [blockId]);
     }
 
+    /**
+     * Gets EXPERIMENTAL_genesis_config from RPC
+     * @returns {Promise<GenesisConfig>}
+     */
+    async genesisConfig(): Promise<GenesisConfig> {
```

Comment:
let's call this `experimental_genesisConfig` to indicate that API isn't final and can be changed